### PR TITLE
Adds the art gallery to Deltastation

### PR DESCRIPTION
**Pull request by Mickyan**
A place to store and share your masterpieces and/or crude drawings of genitalia for the world to see. Paintings placed here will persist between rounds.

:cl:
tweak: Added the art gallery to Deltastation
/:cl:

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
Time to draw cocks and pineheels.

File: code/game/objects/structures/artstuff.dm [hunk 1]
```diff
@@ -354,3 +354,43 @@
 	for(var/x in 1 to width)
 		for(var/y in 1 to height)
 			grid[x][y] = I.GetPixel(w-x,h-y)
+
+//Presets for art gallery mapping, for paintings to be shared across stations
+/obj/structure/sign/painting/library1
+	persistence_id = "library1"
```
Comment (AnturK) [inline code/game/objects/structures/artstuff.dm]: Absolutely no reason to split it into that many categories. Just library, library_secure and library_private is more than enough.

Comment (Mickyan) [reply code/game/objects/structures/artstuff.dm]: Are there no side effects from having multiple frames with the same persistence_id

Comment (AnturK) [reply code/game/objects/structures/artstuff.dm]: No, it's the intended usage.

Comment (Mickyan) [reply code/game/objects/structures/artstuff.dm]: Neat!

**Comment by spessbandit** [comment]
Will framed photographs also persist? If not they should.

**Comment by cacogen** [comment]
I don't want to shit on your efforts but those floor coverings don't look right. Can I ask how the persistence works? Can you take down the paintings? Can they be modified? Can anyone change what paintings are up?

**Comment by Mickyan** [comment]
> I don't want to shit on your efforts but those floor coverings don't look right
This isn't helpful

**Comment by kriskog** [comment]
merge conflict

**Comment by Mickyan** [comment]
nailed it

**Comment by cacogen** [comment]
Look I apologise for being tactful what I meant to say is those floor coverings look bad and I hope they get changed sometime soon

**Pull request merged**